common/thorhelper/thorfile.cpp

```diff
@@ -25,7 +25,7 @@
 #include "rtlfield.hpp"
 #include "rtlds_imp.hpp"
 
-namespace thorfile {
+namespace thorfile {  // Make sure we can't clash with generated versions or version check mechanism fails.
```

Comment:
trivial: for consistency we should have the '{' on the next line (same for examples below)